src/system/line_stream.c

```diff
@@ -111,7 +111,7 @@ char *line_stream_collect_n_lines(LineStream *line_stream, size_t n)
             return NULL;
         }
 
-        result = string_append(result,line);
+        result = string_append(result, line);
```

Comment:
🤔 ... or "fix formatting" should be a separate thing?

Reply:
Nah, not necessarily.